ApplicationLibCode/ProjectDataModel/WellLog/RimWellLogCurveCommonDataSource.cpp

```diff
@@ -838,6 +848,13 @@ void RimWellLogCurveCommonDataSource::fieldChangedByUi( const caf::PdmFieldHandl
     RimWellLogPlot* parentPlot = nullptr;
     this->firstAncestorOrThisOfType( parentPlot );
 
+    if ( changedField == &m_wellPath )
```

Comment:
Consider one if-statement (changedFiled ==&m_wellPath && (m_wellPath() == m_refWellPath())

Reply:
Initially followed the structure of existing if-statements, but one if-statement is better. Updated the previously existing cases in method to single if-statement checks as well.